tests/ssh_test.sh

```diff
@@ -4,7 +4,7 @@
 # NOTE: We're not testing the ssh command here, just the kw ssh operation
 #
 
-. ./src/vm.sh --source-only
+. ./src/remote.sh --source-only
```

Comment:
In this commit, you must move all tests related to `ssh` to the remote file.

Reply:
Ok, this would mean moving the tests from `ssh_test.sh` to `remote_test.sh` and then deleting `ssh_test.sh`. Correct?

Reply:
Yes.